src/guides/bioinformatics/filtering-and-subsampling.rst

```diff
@@ -23,20 +24,101 @@ different types of analysis. A simple example use of this command would be
      --output-sequences filtered_sequences.fasta \
      --output-metadata filtered_metadata.tsv
 
-This command will select all sequences with collection date in 2012 or later.
-The filter command has a large number of options that allow flexible filtering
-for many common situations. One such use-case is the exclusion of sequences that
-are known to be outliers (e.g. because of sequencing errors, cell-culture
-adaptation, ...). These can be specified in a separate text file (e.g.
-``exclude.txt``):
+There are several options that allow flexible filtering for many common
+situations. Options can be divided into the following categories:
```

Comment:
[Not blocking]

This reads as 4 exclusive categories, whereas it's two sets of categories (as demonstrated by the 2x2 table). Consider:

Options can be classified based on what data they operate on: {metadata-based, sequence-based}

as well as their behaviour: {standard, force-inclusive}